components/dropdown/dropdown-content-mixin.js

```diff
@@ -11,7 +11,8 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 	static get properties() {
 		return {
 			/**
-			 * Optionally align dropdown to either `start` or `end`. If not set, the dropdown will attempt be centred.
+			 * Optionally align dropdown to either start or end. If not set, the dropdown will attempt be centred.
+			 * @type {('start'|'end')}
```

Comment:
This is a weird attribute eh... the default is `undefined` which centres it? Hmm... 🤔

Reply:
Yeah - we could add a 'centered' and do a bit of restructuring